CMakeLists.txt

```diff
@@ -82,12 +88,14 @@ message(STATUS "Unit tests: ${ENABLE_TESTS}")
 message(STATUS "Build with -Werror: ${ENABLE_WERROR}")
 
 if (INTROSPECTION_FOUND)
-	message(STATUS "Build GObjectIntrospection: YES")
-	if (VALA_COMPILER)
-		message(STATUS "Build Vala bindings: YES")
-	else()
-		message(STATUS "Build Vala bindings: NO")
-	endif()
+    message(STATUS "Build GObjectIntrospection: YES")
+    if (VALA_COMPILER)
+        message(STATUS "Build Vala bindings: YES")
+    else()
+        message(STATUS "Build Vala bindings: NO")
+    endif()
 else()
-	message(STATUS "Build GObjectIntrospection: NO")
-endif()
\ No newline at end of file
+    message(STATUS "Build GObjectIntrospection: NO")
```

Comment:
As you are already fiddlinging with status messages, how about adding a "Build with Lomiri features" message?